backend/de.metas.adempiere.adempiere/base/src/test/java/de/metas/impexp/format/ImpFormatColumnTest.java

```diff
@@ -0,0 +1,52 @@
+package de.metas.impexp.format;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.sql.Timestamp;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class ImpFormatColumnTest
+{
+	private static final Timestamp EXPECTED_TIMESTAMP = Timestamp.valueOf("2025-03-21 00:00:00");
+	private ImpFormatColumn impFormatColumn;
+
+	@BeforeEach
+	void setUp()
+	{
+		impFormatColumn = ImpFormatColumn.builder()
+				.columnName("testColumn")
+				.dataType(ImpFormatColumnDataType.Date)
+				.dataFormat("dd.MM.yy")
+				.build();
+	}
+
+	@ParameterizedTest
+	@ValueSource(strings = { "21.03.25" })
+	void testParseCellValue_TwoDigitYear_ShouldParseTo2025(final String testDate) throws Exception
```

Comment:
no need for a parametrized test when u have only one test value!

Reply:
and btw, isn't this test covered by testParseCellValue_HandlesBothTwoAndFourDigitYears?

Reply:
yes, removed